funding/manager.go

```diff
@@ -4077,12 +4085,16 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 	}
 }
 
+// handleWarningMsg processes the warning which was received from remote peer.
+func (f *Manager) handleWarningMsg(peer lnpeer.Peer, msg *lnwire.Warning) {
```

Comment:
i think this should come with a test

Reply:
Added `don't fail the channel if we receive a warning message` to `TestChannelLinkFail`